lib/jekyll/scholar/tags/bibliography_count.rb

```diff
@@ -0,0 +1,53 @@
+module Jekyll
+  class Scholar
+
+    class BibliographyCountTag < Liquid::Tag
+      include Scholar::Utilities
+
+      def initialize(tag_name, arguments, tokens)
+        super
+
+        @config = Scholar.defaults.dup
+
+        optparse(arguments)
+      end
+
+      def render(context)
+        set_context_to context
+
+        # Add bibtex files to dependency tree
+        if context.registers[:page] and context.registers[:page].key? "path"
+          bibtex_paths.each do |bibtex_path|
```

Comment:
We should move this code to `utilities` so that both `bibliography` and `bibliography_count` can call it.